src/features/Tutorial/TutorialConfig.jsx

```diff
@@ -88,19 +91,19 @@ export default function Configuration({ simConfig, benchSettings, blocklyConfig,
     form.resetFields();
   };
 
-  const downloadXmlFile = () => {
+  const downloadXmlFileAccount = () => {
     const startIndex = xml.indexOf('<block type="robot_execute"');
```

Comment:
might need to add setError("") and setSuccess("")  to empty warning for the new warning / success message. Otherwise the last warning will display